VagrantBuild.sh

```diff
@@ -34,10 +36,18 @@ fi
 
 echo "Building Hoot... "
 echo "Will take several extra minutes to build the training data the initial time Hootenanny is installed only."
-make -s clean && make -sj$(nproc)
+#make -s clean && make -sj$(nproc)
+make -s clean && make -sj3
```

Comment:
If I remember right, we upped the memory to 10 GB so that compilation wouldn't fail with `make -sj$(nproc)` (4) instead of (3).  Is that not the case?  See `Vagrantfile` line 140 & 141.

Reply:
Fixed.